### ISSUE TITLE
IP forwarding doesn't work on Buster

### ISSUE THREAD

**Issue by martignoni** [moodlebox/moodlebox#133]
On buster, `nftables` is the default for packet filtering and NAT. Current rules break IP forwarding.

See this example for a solution: https://superuser.com/questions/985800/complete-masquerading-nat-example-using-nftables-on-linux

**Comment by martignoni** [comment]
Use https://wiki.nftables.org/wiki-nftables/index.php/Moving_from_iptables_to_nftables to translate rules.

**Comment by martignoni** [comment]
New ruleset:
```
add table ip nat
add chain ip nat PREROUTING { type nat hook prerouting priority -100; policy accept; }
add chain ip nat INPUT { type nat hook input priority 100; policy accept; }
add chain ip nat OUTPUT { type nat hook output priority -100; policy accept; }
add chain ip nat POSTROUTING { type nat hook postrouting priority 100; policy accept; }
add rule ip nat POSTROUTING oifname "eth0" counter masquerade 
add table ip filter
add chain ip filter INPUT { type filter hook input priority 0; policy accept; }
add chain ip filter FORWARD { type filter hook forward priority 0; policy accept; }
add chain ip filter OUTPUT { type filter hook output priority 0; policy accept; }
add rule ip filter FORWARD iifname "eth0" oifname "wlan0" ct state related,established  counter accept comment "MoodleBox forwarding rules"
add rule ip filter FORWARD iifname "wlan0" oifname "eth0" counter accept comment "MoodleBox forwarding rules"
```

**Comment by martignoni** [comment]
See https://www.reddit.com/r/debian/comments/6yojtv/debian_9_how_to_setup_nftables_fail2ban/

**Comment by martignoni** [comment]
We'll stick to normal iptables syntax for now. Problem was with iptables-persistent.

Fixed by ab328e371133334a6159a75300829aae49b9f16e. Closing.
